packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -128,7 +129,7 @@ export const ClayCardWithNavigation = ({
 					{horizontal && (
 						<ClayCard.Row>
 							<ClayLayout.ContentCol>
-								<ClaySticker inline>
+								<ClaySticker inline title={horizontalSymbol}>
```

Comment:
We can add a new API here so that the developer can add the title to the sticker, something like `stickerTitle` would be enough.

Reply:
Hi @matuzalemsteles,
For this comment and your next comment, do you have any examples of how to do this?

Reply:
@SelenaAungst you first have to declare the type of `stickerTitle` in the component for example https://github.com/liferay/clay/blob/40e3d5c8f70bbcecf942314d1b6bd232e97c6b2b/packages/clay-card/src/CardWithInfo.tsx#L101 and later just declare the `stickerTitle` in the component props so you can use https://github.com/liferay/clay/blob/40e3d5c8f70bbcecf942314d1b6bd232e97c6b2b/packages/clay-card/src/CardWithInfo.tsx#L135.

Reply:
@matuzalemsteles 
Thank you! I think I was able to make all the changes you requested and get everything working as intended. I will send a new PR for this and mention it here.